mxcube3/__init__.py

```diff
@@ -158,6 +161,7 @@ def complete_initialization(app):
         except Exception:
             sys.excepthook(*sys.exc_info())
 
+        logging.getLogger("HWR").info("Mxcube3 server initialized, it took %.1f seconds" % (time.time() - t0))
```

Comment:
I see much longer time now in the beamline, ~35 s. But, we also have more hwobj in place so I cannot tell if there is something around mxcube3 or just Maxiv hwobjs. Also, you never know which is the last step of the process, so a msg at the end seems useful.